src/build/__init__.py

```diff
@@ -161,6 +163,17 @@ def __init__(self, srcdir='.', config_settings=None, python_executable=sys.execu
             python_executable=python_executable,
         )
 
+    @property
+    def python_executable(self):  # type: () -> Union[bytes, Text]
+        # make mypy happy
+        if not isinstance(self.hook.python_executable, typing.Text) and not isinstance(self.hook.python_executable, bytes):
```

Comment:
I think you can replace this with ``isinstance(self.hook.python_executable, (typing.Text, bytes))`` 👍 no need for the and here 🤔

Reply:
I missed that in the `isinstance` documentation :upside_down_face: